cmd/access/access_node_builder.go

```diff
@@ -41,20 +53,92 @@ type AccessNodeBuilder interface {
 	ParticipatesInUnstakedNetwork() bool
 }
 
+type AccessNodeConfig struct {
+	staked                       bool
+	stakedAccessNodeIDHex        string
+	unstakedNetworkBindAddr      string
+	blockLimit                   uint
+	collectionLimit              uint
+	receiptLimit                 uint
+	collectionGRPCPort           uint
+	executionGRPCPort            uint
+	pingEnabled                  bool
+	nodeInfoFile                 string
+	apiRatelimits                map[string]int
+	apiBurstlimits               map[string]int
+	FollowerState                protocol.MutableState
+	IngestEng                    *ingestion.Engine
+	RequestEng                   *requester.Engine
+	FollowerEng                  *followereng.Engine
+	SyncCore                     *synchronization.Core
+	rpcConf                      rpc.Config
+	RpcEng                       *rpc.Engine
+	FinalizationDistributor      *pubsub.FinalizationDistributor
+	CollectionRPC                access.AccessAPIClient
+	ExecutionNodeAddress         string // deprecated
+	HistoricalAccessRPCs         []access.AccessAPIClient
+	ConCache                     *buffer.PendingBlocks // pending block cache for follower
+	TransactionTimings           *stdmap.TransactionTimings
+	CollectionsToMarkFinalized   *stdmap.Times
+	CollectionsToMarkExecuted    *stdmap.Times
+	BlocksToMarkExecuted         *stdmap.Times
+	TransactionMetrics           module.TransactionMetrics
+	PingMetrics                  module.PingMetrics
+	logTxTimeToFinalized         bool
+	logTxTimeToExecuted          bool
+	logTxTimeToFinalizedExecuted bool
+	retryEnabled                 bool
+	rpcMetricsEnabled            bool
+}
+
+func DefaultAccessNodeConfig() *AccessNodeConfig {
+	return &AccessNodeConfig{
+		receiptLimit:       1000,
```

Comment:
these defaults were earlier defines as part of the definition of the command line args. For the untaked AN, there will be no command line args, hence the definition of the default values here as the `DefaultAccessNodeConfig`